Update meta.yaml and run `conda smithy rerender`

wd15 opened the pull request:
See https://github.com/conda-forge/conda-forge.github.io/issues/303

wd15 committed:
Update meta.yaml and run `conda smithy rerender`

See https://github.com/conda-forge/conda-forge.github.io/issues/303

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

isuruf commented:
It seems you need scipy in build requirements.

wd15 committed:
Add scipy to build

Due to https://circleci.com/gh/conda-forge/sfepy-feedstock/2?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

isuruf commented:
You need matplotlib, pyparsing and pytables in build requirements

wd15 committed:
Updating build requirements

Adding requirements that were not requied in previous phases of conda
forge testing.

isuruf commented on recipe/meta.yaml:
```diff
@@ -13,13 +13,16 @@ source:
 
 build:
   number: 0
-  skip: true
```
Can you make this `skip: true   # [py>35]` to skip python3.6 as not all your dependencies are on python3.6 yet?

isuruf replied:
You'll need to rerender after that

wd15 committed:
MNT: Re-rendered with conda-smithy 1.7.0

wd15 committed:
Skipping python 3.6

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form.

wd15 committed:
Fix spacing for linter

wd15 commented:
@isuruf it looks like things have passed, shall I go ahead and merge?

isuruf reviewed (approved):
LGTM
